sensu/plugins/check-large-receive-offload.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+#
+# Checks that large receive offload is disabled
+#
+# Return CRITICAL if large receive offload is enabled
+#
+# Dean Daskalantonakis <ddaskal@us.ibm.com>
+
+import argparse
+import re
+import subprocess
+import sys
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+
+
+def exit_with_error_status(critical):
+    if critical:
+        sys.exit(STATE_CRITICAL)
+    else:
+        sys.exit(STATE_WARNING)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--interface', help='primary interface')
+    parser.add_argument('-z', '--criticality', default='CRITICAL')
+    args = parser.parse_args()
+
+    if args.criticality.upper() == 'CRITICAL':
```

Comment:
alright well since it's currently being passed in as a string, I need to make a follow-up PR to ursula in order to have it passed in as boolean instead. do you want me to go down that route?